src/components/Tool/Appeal/Flow/ContactFlowRow/ContactFlowRow.tsx

```diff
@@ -1,5 +1,6 @@
 import React, { useEffect, useMemo } from 'react';
```

Comment:
We need to make the "Select all _ contacts" button work.
<img width="816" alt="Screenshot 2024-08-05 at 12 13 58 PM" src="https://github.com/user-attachments/assets/9ed4440f-db62-428d-a9eb-b0b880797de9">

Reply:
That is what I'm going to add at a later time